src/components/UI/VersionPicker/__tests__/VersionPicker.js

```diff
@@ -0,0 +1,124 @@
+import '@testing-library/jest-dom/extend-expect';
+
+import { fireEvent, render, within } from '@testing-library/react';
+import { ThemeProvider } from 'emotion-theming';
+import React from 'react';
+import theme from 'styles/theme';
+import AppVersionPicker from 'UI/VersionPicker/VersionPicker';
+
+it('renders without crashing', () => {
+  render(
+    <ThemeProvider theme={theme}>
```

Comment:
You can use the `renderWithTheme()` helper in `testUtils/renderUtils` if you don't want to add `ThemeProvider` in every test 👍